ngraph/python/src/ngraph/ops/opset3/__init__.py

```diff
@@ -0,0 +1,157 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+"""ngraph module namespace, exposing factory functions for all ops and other classes."""
+# noqa: F401
+
+from pkg_resources import get_distribution, DistributionNotFound
+
+try:
+    __version__ = get_distribution("ngraph-core").version
+except DistributionNotFound:
+    __version__ = "0.0.0.dev0"
+
+
+from ngraph.ops.opset1 import absolute
+from ngraph.ops.opset1 import absolute as abs
+from ngraph.ops.opset1 import acos
+from ngraph.ops.opset1 import add
+from ngraph.ops.opset1 import asin
+from ngraph.ops.opset3 import assign
+from ngraph.ops.opset1 import atan
+from ngraph.ops.opset1 import avg_pool
+from ngraph.ops.opset1 import batch_norm_inference
+from ngraph.ops.opset2 import batch_to_space
+from ngraph.ops.opset1 import binary_convolution
+from ngraph.ops.opset3 import broadcast
+from ngraph.ops.opset3 import bucketize
+from ngraph.ops.opset1 import ceiling
+from ngraph.ops.opset1 import ceiling as ceil
+from ngraph.ops.opset1 import clamp
+from ngraph.ops.opset1 import concat
+from ngraph.ops.opset1 import constant
+from ngraph.ops.opset1 import convert
+from ngraph.ops.opset1 import convert_like
+from ngraph.ops.opset1 import convolution
+from ngraph.ops.opset1 import convolution_backprop_data
+from ngraph.ops.opset1 import cos
+from ngraph.ops.opset1 import cosh
+from ngraph.ops.opset1 import ctc_greedy_decoder
+from ngraph.ops.opset3 import cum_sum
+from ngraph.ops.opset3 import cum_sum as cumsum
+from ngraph.ops.opset1 import deformable_convolution
+from ngraph.ops.opset1 import deformable_psroi_pooling
+from ngraph.ops.opset1 import depth_to_space
+from ngraph.ops.opset1 import detection_output
+from ngraph.ops.opset1 import divide
+from ngraph.ops.opset1 import elu
+from ngraph.ops.opset3 import embedding_bag_offsets_sum
+from ngraph.ops.opset3 import embedding_bag_packed_sum
+from ngraph.ops.opset3 import embedding_segments_sum
+from ngraph.ops.opset3 import extract_image_patches
+from ngraph.ops.opset1 import equal
+from ngraph.ops.opset1 import erf
+from ngraph.ops.opset1 import exp
+from ngraph.ops.opset1 import fake_quantize
+from ngraph.ops.opset1 import floor
+from ngraph.ops.opset1 import floor_mod
+from ngraph.ops.opset1 import gather
+from ngraph.ops.opset1 import gather_tree
+from ngraph.ops.opset2 import gelu
+from ngraph.ops.opset4 import get_output_element
```

Comment:
Probably we shouldn't import op from higher opset, so `opset4` in `opset3`

Reply:
Done